### ISSUE TITLE
interpretation NODDI output

### ISSUE THREAD

**Issue by SanRut** [robbert-harms/MDT#31]
Dear Robbert,

I've fitted the NODDI model and I'm uncertain about the interpretation of the various output maps. I'm specifically looking for the correspondence between the outputs w_ic.w, w_ec.w & w_csf.w and the NODDI formula from the Zhang 2012 paper*, where you also refer to in your NeuroImage (2017) paper. 

I'm quoting the formula from the Zhang paper below (unfortunately I can not use subscript):

A = (1 - **V**iso) (**V**ic **A**ic + (1-**V**ic)**A**ec) + **V**iso **A**iso
They further describe **V**ic as representing: neurite density / volume fraction of intra cellular compartment

I assumed that the w_ic.w from the MDT corresponds to **V**ic, and thus representing the neurite density. However given that w_ic.w+w_ec.w+w_csf.w = 1, I'm not so sure anymore. 

So I'm wondering where do the three maps: w_ic.w, w_ec.w & w_csf.w correspond to in the Zhang formula, and how should I interpret the three maps respectively. 

Also, in your NeuroImage paper it is written that the Orientation Dispersion Index (ODI) is defined in the Zhang 2011** paper. I failed to find the corresponding formula and given the output-values of the ODI-maps, I was wondering whether it could be the Zhang 2012 paper? 

Thanks a lot for your answers (again) and best of wishes, Sanne

** Zhang, H., Hubbard, P.L., Parker, G.J., Alexander, D.C., 2011. Axon diameter mapping in
the presence of orientation dispersion with diffusion MRI. Neuroimage 56 (3),
1301–1315.
* Zhang, H., Schneider, T., Wheeler-Kingshott, C.A., Alexander, D.C., 2012. NODDI:
practical in vivo neurite orientation dispersion and density imaging of the human
brain. Neuroimage 61 (4), 1000–1016.

**Comment by robbert-harms** [comment]
Hi Sanne,

All models in MDT are configured as a weighted sum of one or more compartments. As such, the NODDI model in MDT is written as (using your notation:

A = **V**iso **A**iso + **V**ic **A**ic + **V**ec **A**ec
with the constraint that the weights sum to 1.

In my language, this is:
S = w_iso.w*S_ico + w_ic.w * S_ic + w_ec.w * S_ec

If you would work it out, you can see that these definitions are equal to each other. 

Then comes the point that Zhang describes the NDI as the **V**ic. Due to linearizing the model definition, the NDI in MDT needs to be computed as NDI = w_ic.w / (w_ic.w + w_ec.w). 

In short, there is no direct correspondence between the volume fraction definitions given in Zhang and those in MDT, although with some algebra you can arrive at the same definitions.

About the ODI definition, you may be right, and that I cited the wrong article.

**Comment by SanRut** [comment]
Hi Robbert,

Thanks a lot for your prompt reply, it all makes way more sense. I was also struggling with the NDI definition indeed. Your answer is detailed enough. 

Best, Sanne

**Comment by robbert-harms** [comment]
You are welcome. Let me know if there is anything else.

Best, Robbert

**Comment by shaanxicode** [comment]
Hi Robbert,

I am wondering where is the w_iso.w? Is it the w_csf.w? If so, I found it is different from the output calculated by Matlab toolbox, offered by Zhang 2012 paper. When I calculated the HCP S1200 data.

Best, Shu

**Comment by robbert-harms** [comment]
Hi Shu,

Yes, w_csf.w is the w_iso.w. What kind of differences do you experience?

Some small differences may be possible due to the model reformulation and different optimization methods. When in doubt, compare the log likelihood values to see which model approximates the results better.

Best,

Robbert

**Comment by shaanxicode** [comment]
Hi Robbert,

Sorry for replying so late. There's something wrong with my notification. And your answer made sense. Thanks a lot for your reply.

Best, Shu
